scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<ul><li>[minor] typo. Think you mean <code>TrackingDataset</code> rather than TorchvisionDataset (not that I put the TrackingDataset inside ` since it's code)</li><li>Here and other places: internal implementation details can be hidden from the user, e.g. "wrapper in a way that can recognized by the FairMOT repo code"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>